Client/include/Room.h

```diff
@@ -37,12 +38,17 @@ namespace Ui {
     struct Room;
 }
 
+enum class RoomType {
+    Summer,
+    Strange
+};
+
 struct Room : public QWidget {
     Q_OBJECT
 
 public:
     Room(Client *client_, Player *player_, QVector<PlayerView *> &players_,
-         QWidget *parent = nullptr);
+         RoomType r_type, QWidget *parent = nullptr);
```

Comment:
```suggestion
         RoomType room_type, QWidget *parent = nullptr);
```

И в местах использования

Reply:
fixed